src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -244,52 +244,50 @@ describe('Language selection', () => {
 
   test('translatable fields appear in selected languages', async () => {
     const { user } = renderComponent();
+    await actWait();
     const languageSelector = await screen.findByTestId(
       formLanguageSelectorTestId
     );
-
     // Select Swedish (with Finnish that is already selected)
     const sv = await within(languageSelector).findByRole('checkbox', {
       name: /ruotsi/i,
     });
     await user.click(sv);
+    await actWait();
     await waitFor(() => {
       expect(sv).toBeChecked();
     });
 
     for (const labelText of translatableFieldLabels) {
-      await waitFor(
-        () => {
-          expect(
-            screen.getByLabelText(
-              new RegExp(labelText.source + / \(FI\)/.source)
-            )
-          ).toBeInTheDocument();
-          expect(
-            screen.getByLabelText(
-              new RegExp(labelText.source + / \(SV\)/.source)
-            )
-          ).toBeInTheDocument();
-        },
-        { timeout: 5_000 }
-      );
+      await waitFor(() => {
+        expect(
+          screen.getByLabelText(new RegExp(labelText.source + / \(FI\)/.source))
+        ).toBeInTheDocument();
+      });
+      await waitFor(() => {
+        expect(
+          screen.getByLabelText(new RegExp(labelText.source + / \(SV\)/.source))
+        ).toBeInTheDocument();
+      });
```

Comment:
A trivial thing, but the waitFor should not be needed twice here. 

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#having-multiple-assertions-in-a-single-waitfor-callback

Reply:
Shouldn't hurt either to have two waitFor clauses that each have only one expect clause in them. The article you linked was about "Having multiple assertions in a single waitFor callback" which is not applicable here as there is only one assertion in each waitFor callback.

Reply:
But I did have cases where there were more than one expect clause inside a single "await waitFor", changed those in 969e75148149d1da502c1941af8a597f2dfa197c to use only a single expect clause per "await waitFor".